modules/catalog/src/main/java/org/apache/ignite/internal/catalog/events/AlterColumnEventParameters.java

```diff
@@ -23,32 +23,36 @@
  * Create table event parameters contains a column descriptor for the modified column.
  */
 public class AlterColumnEventParameters extends CatalogEventParameters {
-
     private final int tableId;
 
-    private final CatalogTableColumnDescriptor columnDescriptor;
+    private final CatalogTableColumnDescriptor descriptor;
 
     /**
      * Constructor.
      *
      * @param causalityToken Causality token.
-     * @param tableId Returns an id the table to be modified.
-     * @param columnDescriptor Descriptor for the column to be replaced.
+     * @param catalogVersion Catalog version.
+     * @param tableId Returns ID the table to be modified.
+     * @param descriptor Descriptor for the column to be replaced.
      */
-    public AlterColumnEventParameters(long causalityToken, int tableId, CatalogTableColumnDescriptor columnDescriptor) {
-        super(causalityToken);
+    public AlterColumnEventParameters(long causalityToken, int catalogVersion, int tableId, CatalogTableColumnDescriptor descriptor) {
+        super(causalityToken, catalogVersion);
 
         this.tableId = tableId;
-        this.columnDescriptor = columnDescriptor;
+        this.descriptor = descriptor;
     }
 
-    /** Returns an id of a modified table. */
+    /**
+     * Returns ID of a modified table.
+     */
```

Comment:
Was this necessary? Do we have code style guide that recommends this formatting?
If not, then you simply change someone-else's code by your own preference, without any other objective reason.
I wouldn't mind it, but this PR is 1000+ lines of changes, and some of them are completely unnecessary. This forces me to look at more code than I should.

Reply:
It is grammatically correct to use ID.

Reply:
Ok, but converting
```
/** Foo. */
```
into
```
/**
 * Foo.
 */
```
is not related to grammar